roles/gitea-ocp/templates/service.yml.j2

```diff
@@ -13,5 +13,11 @@ spec:
     port: 3000
     protocol: TCP
     targetPort: 3000
+  - name: gitea-ssh
+    port: 2022
+    protocol: TCP
+    targetPort: 2022
   sessionAffinity: None
-  type: ClusterIP
+{% if _gitea_external_ssh|bool %}
+  type: LoadBalancer
```

Comment:
Since this also changes the ClusterIP service for HTTP to LoadBalancer, should this be split into a separate service?